sdk/storage/azure-storage-common-data-movement/src/main/java/com/azure/storage/common/data/movement/PathScanner.java

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.common.data.movement;
+
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+import reactor.core.scheduler.Schedulers;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.*;
```

Comment:
In Java, generally we like to flatten the imports to just the minimal set, that way we aren't pulling in classes that are unused.

I think there is an Intellij setting for this to prevent it from flattening for you. https://stackoverflow.com/questions/26225989/how-do-i-stop-wildcard-java-package-imports-in-intellij